build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
         google()
         jcenter()
         mavenCentral()
+        maven { url 'https://dl.bintray.com/android/android-tools/' }
```

Comment:
Nice fix

Reply:
Thanks, weird that it changed all of a sudden like that...